cloudinit/config/cc_resolv_conf.py

```diff
@@ -26,12 +26,18 @@
 }
 
 MODULE_DESCRIPTION = """\
+Unless manually editing :file:`/etc/resolv.conf` is the correct way to manage
+nameserver information on your operating system, you do not want to use
+this module. Many distros have moved away from manually editing ``resolv.conf``
+so please verify that this is the preferred nameserver management method for
+your distro :file:`/etc/resolv.conf` before using this module.
```

Comment:
```suggestion
your distro before using this module.
```

Reply:
Reading this sentence aloud, I got a bit lost in it, and I think this is the offending party